### PR TITLE
Increase the annotation size limit to 256k

### PR DESCRIPTION
**Pull request by janetkuo**
Address https://github.com/kubernetes/kubernetes/issues/15878#issuecomment-149704721

cc @bgrant0607 @jackgr @liggitt

### CODE REVIEW

**Comment by lavalamp** [comment]
I think it's OK. But it means if you upgrade to 1.1, make a thing with a huge annotation, and downgrade to 1.0, you'll have stored objects that don't pass validation.

That seems exceedingly unlikely.

LGTM

**Comment by bgrant0607** [comment]
In general, we don't deal well with rolling back after having upgraded a cluster. I agree we need to deal with that eventually, but we certainly aren't maintaining that discipline today.

File: pkg/api/validation/validation_test.go [hunk 1]
```diff
@@ -191,10 +191,10 @@ func TestValidateAnnotations(t *testing.T) {
 		{"1234/5678": "bar"},
 		{"1.2.3.4/5678": "bar"},
 		{"UpperCase123": "bar"},
-		{"a": strings.Repeat("b", (64*1024)-1)},
+		{"a": strings.Repeat("b", (256*1024)-1)},
 		{
-			"a": strings.Repeat("b", (32*1024)-1),
-			"c": strings.Repeat("d", (32*1024)-1),
+			"a": strings.Repeat("b", (128*1024)-1),
```
Comment (bgrant0607) [inline pkg/api/validation/validation_test.go]: Could we use totalAnnotationSizeLimitB here?

**Comment by bgrant0607** [comment]
Thanks much.

File: pkg/api/validation/validation_test.go (hunk 1 above)
Comment (JanetKuo) [inline pkg/api/validation/validation_test.go]: Fixed

**Comment by jackgr** [comment]
Thanks, Janet.

**Pull request merged**